src/main/scala/io/iohk/ethereum/crypto/ECDSASignature.scala

```diff
@@ -95,22 +39,72 @@ object ECDSASignature {
     ECDSASignature(r, s, v)
   }
 
-}
+  /**
+    * new formula for calculating point sign post EIP 155 adoption
+    * v = CHAIN_ID * 2 + 35 or v = CHAIN_ID * 2 + 36
+    */
```

Comment:
@mcsherrylabs there is no block number for this
https://github.com/ethereum/EIPs/issues/155
it was just implemented, so for me it should be enabled by default for all blocks
also there is PR for yellow paper
https://github.com/ethereum/yellowpaper/pull/282/files